pkg/api/blackduck/v1/types.go

```diff
@@ -73,6 +73,10 @@ type BlackduckSpec struct {
 	ImageUIDMap           map[string]int64          `json:"imageUidMap,omitempty"`
 	LicenseKey            string                    `json:"licenseKey,omitempty"`
 	RegistryConfiguration RegistryConfiguration     `json:"registryConfiguration,omitempty"`
+	AdminPassword         string                    `json:"adminPassword"`
+	UserPassword          string                    `json:"userPassword"`
+	PostgresPassword      string                    `json:"postgresPassword"`
+	BlackduckPassword     string                    `json:"blackduckPassword"`
```

Comment:
whether the blackduck password to be removed from the struct?

Reply:
No, because opssight still use BlackduckSpec: `blackduckPassword, _ := util.Base64Decode(opssight.Spec.Blackduck.BlackduckSpec.BlackduckPassword)`

Reply:
Done.